CHANGES_NEXT_RELEASE

```diff
@@ -28,3 +28,4 @@
 - Fix: Empty string detection for entity id, entity type and attribute name in NGSIv2 request URLs (#1426, #1487)
 - Fix: Trailing slashes in URI path no longer silently removed, but considered and giving adequate error responses
 - Fix: Avoid rendering at the same time id and idPattern fields in NGSIv2 subscriptions
+- Fix: Error returned on GET/DELETE requests specifying Content-Type (Issue #2128)
```

Comment:
Probably the right one to mention #1887 is:

* Fix: out-of-NGSI service not found error now using 400 Bad Request HTTP response code (instead of 200 OK). Error message has been unified to "service not found" in all cases.